testing/go/create_table_test.go

```diff
@@ -159,3 +159,63 @@ func TestCreateTable(t *testing.T) {
 		},
 	})
 }
+
+func TestCreateTableInherit(t *testing.T) {
+	RunScripts(t, []ScriptTest{
+		{
+			Name: "Create table with inheritance",
+			SetUpScript: []string{
+				"create table t1 (a int);",
+				"create table t2 (b int);",
+				"create table t3 (c int);",
+				"create table t11 (a int);",
+			},
+			Assertions: []ScriptTestAssertion{
+				{
+					Query:    "create table t4 (d int) inherits (t1, t2, t3);",
+					Expected: []sql.Row{},
+				},
+				{
+					Query:    "insert into t4 values (1, 2, 3, 4);",
```

Comment:
Make sure the column order is as expected (insert with all columns named, then select *)